presto-spi/pom.xml

```diff
@@ -53,6 +53,12 @@
             <artifactId>jol-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
We never do this :) There is a very involved reason for this. I believe that you need this for `ResourceGroupQueryLimits#toString` - I have recommended another suggestion there.